src/lit-element.ts

```diff
@@ -195,13 +195,17 @@ export class LitElement extends UpdatingElement {
   protected update(changedProperties: PropertyValues) {
     super.update(changedProperties);
     const templateResult = this.render() as unknown;
+
     if (templateResult instanceof TemplateResult) {
-      (this.constructor as typeof LitElement)
+      return;
```

Comment:
Did you mean to negate the instanceof check above?

Reply:
Yes!
Wow, that 2nd commit didn't get pushed.